src/guards/role.guard.ts

```diff
@@ -63,8 +56,8 @@ export class RoleGuard implements CanActivate {
     // Grab access token from grant
     const accessToken: KeycloakConnect.Token = grant.access_token as any;
 
-    return allowAnyRole
-      ? roles.some(r => accessToken.hasRole(r))
-      : roles.every(r => accessToken.hasRole(r));
+    return rolesMetaData.RoleMatchingMode === RoleMatchingMode.any
```

Comment:
Should we add any as default or no ? Feel free to suggest or we can leave it as is.

Reply:
Good point, what do you think about making `RoleMatchingMode.any` to default?

Reply:
Sure, but it will break behavior before this, going to add a warning when I release this.